artscraper/find_artworks.py

```diff
@@ -182,13 +185,14 @@ def get_artist_works(self):
 
         # Check if right arrow button can still be clicked
         while right_arrow_element.get_attribute('tabindex') is not None:
+            time.sleep(random_wait_time(min_wait=self.min_wait_time))
             # Find right arrow button
             right_arrow_element = parent_element.find_element('xpath', \
                 './/*[contains(@data-gaaction,"rightArrow")]')
             # Click on right arrow button
             self.driver.execute_script("arguments[0].click();", right_arrow_element)
             # Wait for page to load
-            time.sleep(random_wait_time(min_wait=self.min_wait_time))
+            #time.sleep(random_wait_time(min_wait=self.min_wait_time))
```

Comment:
Why has this been moved? Remove the commented line if this is what we want.

Reply:
I wanted it to wait for the page to load, then click on the right-arrow element, rather than try to find the right-arrow element first and then wait. I thought this would prevent problems the first time it enters the loop, in case it found no right arrow element because it hadn't yet loaded.